---
# Feel free to add content and custom Front Matter to this file.
# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults

layout: page
title: Research 

---


 <img class="container__image" src='./images/My_pic.png' width='150' height='200' align="left">
 <p> <br/>  <br/>  Rhondene Wint <br/> PhD Candidate<br/> Dept. of Quantitative and Systems Biology <br/> University of California-Merced <br/> <b>Interests</b>: RNA, neurobiology, bioinformatics, machine learning   <br/>  <br/>  <br/>  </p>


## Biography
I am currently a 5<sup>th</sup> year PhD candidate in the Quantitative and Systems Biology deparatment at the <a href="https://qsb.ucmerced.edu/">University of California-Merced</a> under the co-mentorship of Dr. <a href="https://qsb.ucmerced.edu/content/michael-cleary">Michael Cleary </a> and Dr. <a href="https://qsb.ucmerced.edu/content/david-ardell">David Ardell </a>. I am also affiliated with the <a href="https://jgi.doe.gov/our-science/science-programs/fungal-genomics/">Fungal Genomics Group</a> (lead PI Dr.Igor Grigoriev) at the Lawrence Berekely National Laboratory <a href="https://jgi.doe.gov/">Joint Genome Institute</a>.
I earned my B.Sc. in Biological Sciences (magna cum laude, 2014) from <a href="https://www.ncu.edu.jm/">Northern Caribbean University </a>, where I also taught introductory biology labs from 2011-2014. From 2014-2017, I taught the equivalent of AP-level biology, physics and chemistry to post-secondary students.

<br/>Here is my  <a href="https://podcasts.google.com/feed/aHR0cHM6Ly9mZWVkcy5zaW1wbGVjYXN0LmNvbS9GRlZldmJfUg/episode/Zjk1NTAzMmUtNDc4YS00MTRjLWFkOTMtNGYwNmNjNzc1ODBm?sa=X&ved=0CAUQkfYCahcKEwi4tcSehuL0AhUAAAAAHQAAAAAQNQ">guest interview</a> on the podcast Impact Learning hosted by <a href="https://www.linkedin.com/in/mariaxenidou/">Maria Xenidou(PhD)</a> where I discuss my education journey and the power of self-directed learning for future-proofing oneself.

## Research: RNA on the Brain
I employ experimental and computational methods to study RNA biology at the developmental and evolutionary timescales.
<figure>
    <img src='./images/Codon_optimality.png' width='400' height='200'/>
	<figcaption> <b>Codon optimality principle: mRNAs with codons recognized by abundant tRNAs are better translated during protein synthesis.</b>
		</figcaption>
	</figure>

# <font color='brown'> Main Project: Codon optimality and tRNA dynamics during neural differentiation </font>
   ![My helpful screenshot](./images/neuro_header.png) <br/>
A major goal of neurobiology is to unravel the molecular events that orchestrate neural cell-fate and regulation. 
Using genetic manipulation tools and next-generation sequencing, my research investigates how changes in the transfer RNAs (tRNAs), <i> in vivo</i>, between neural progenitors and mature neurons establish their cell-type specific identity and function.
 
	
 <figure>
	<img src='./images/tRNA_research_cartoon.png' width='400' height='200'/>
	<figcaption> <b>My ongoing research on how tRNAs drive post-transcriptional regulation of gene expression in neurodevelopment.</b> 
		</figcaption>	
	</figure>


# <font color='teal'> Evolution of Codon Optimality in Kingdom Fungi/Machine Learning for Codon Optimality </font>
Studies in model fungi agree on codon usage bias as an adaptation for fine-tuning gene expression; however, such knowledge is lacking for most other fungi. 
In collaboration with the JGI Fungal Genomics group, I bioinformatically characterized the codon and tRNA usage of hundreds of species distributed across the 6 major phyla of Kingdom Fungi. 
I employed phylogenetic comparative methods to elucidate the mode and tempo of the macroevolution of codon usage patterns.
 As an application, I went on to develop a <i>sequence-to-expression</i> neural network tool, <b> Codon2Vec </b>, for predicting 
 highly expressed mRNAs based on codon composition (<i>accepted to Mol.Bio&Evo, Dec. 2021</i>). 
<div>
  <figure class="left" style="float:left">
    <img src='./images/fungal_tree.png' width='200' height='200'/>
		<figcaption> <b>Fungal Tree by <a href="https://mycocosm.jgi.doe.gov/mycocosm/home"> JGI/Mycocosm</a> </b>
		</figcaption>
	</figure>
	
  <figure class="left" style="float:right">
		<img src='./images/C2V_schema.png' width='500' height='200'/>
		<figcaption> <b>Codon2Vec schema. Source: <a href="https://github.com/rhondene/Codon2Vec"> R.Wint/Github </a></b>
		</figcaption>
	</figure>

</div>




